src/Foundation/Speedy/code/Services/CriticalGenerationLocal.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Newtonsoft.Json;
+using RestSharp;
+using Sitecore.Foundation.Speedy.Model;
+using Sitecore.Foundation.Speedy.Settings;
+
+namespace Sitecore.Foundation.Speedy.Services
+{
+    public class CriticalGenerationLocal : ICriticalGenerationGateway
+    {
```

Comment:
Might be some confusion here. This is using RestClient to call the remote API hosted on Azure. For production purposes.  It uses a remote API as it requires a none headless browser support which we can't get when node runs in Azure.